lib/radiator_web/live/admin_live/index.ex

```diff
@@ -125,6 +139,20 @@ defmodule RadiatorWeb.AdminLive.Index do
   end
 
   def handle_event("save", %{"show" => params}, socket) do
+    save_show(socket, socket.assigns.action, params)
+  end
+
+  def handle_event("delete", %{"id" => id}, socket) do
+    network = Podcast.get_network!(id)
+    {:ok, _} = Podcast.delete_network(network)
+
+    socket
+    |> assign(:networks, Podcast.list_networks(preload: :shows))
+    # |> stream_delete(:networks, network)}
```

Comment:
why keep this commented line?

Reply:
Weird. I haven't changed the `handle_event("delete"`.